internal/cliutils/packagerepository.go

```diff
@@ -0,0 +1,26 @@
+package cliutils
+
+import (
+	"context"
+
+	"github.com/glasskube/glasskube/api/v1alpha1"
+)
+
+func GetDefaultRepo(ctx context.Context) (*v1alpha1.PackageRepository, error) {
+	var repos v1alpha1.PackageRepositoryList
+	var defaultRepo v1alpha1.PackageRepository
+
+	client := PackageClient(ctx)
+	if err := client.PackageRepositories().GetAll(ctx, &repos); err != nil {
+		return nil, err
+	}
+
+	for _, r := range repos.Items {
+		if r.IsDefaultRepository() {
+			defaultRepo = r
+			break
+		}
+	}
+
+	return &defaultRepo, nil
```

Comment:
This would return an invalid value if there is no default repo. I think we should return an error in that case instead.

Reply:
Done, thanks!